lib/watch/tsc-watcher.js

```diff
@@ -101,4 +104,28 @@ function reportWatchStatusChanged (diagnostic) {
   console.info(ts.formatDiagnostic(diagnostic, formatHost))
 }
 
+function requireTsConfig (configPath) {
+  const isModule = basename(configPath) === configPath
+  let configFile = require(isModule ? configPath : resolve(process.cwd(), configPath))
```

Comment:
I think the tsconfig can be a json5 with comments, right? So this won't work.

Reply:
Ok, I'll add `require('json5/lib/register')`